content/docs/04.6-vm-management/05-vm-packs-profiles/25-create-vm-namespace.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Create a Namespace for VMs"
```

Comment:
As mentioned in a different comment, I don't think we should cover this in the documentation right now.

Reply:
removed